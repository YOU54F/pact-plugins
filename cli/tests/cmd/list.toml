bin.name = "pact-plugin-cli"
args = "list --help"
