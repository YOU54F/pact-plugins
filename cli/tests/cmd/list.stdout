List installed or available plugins

Usage: pact-plugin-cli list <COMMAND>

Commands:
  installed  List installed plugins
  known      List known plugins
  help       Print this message or the help of the given subcommand(s)

Options:
  -h, --help  Print help
